### PR TITLE
Fix build for LDC 1.38

### PR DESCRIPTION
**Pull request by JohanEngelen**

### CODE REVIEW

**Comment by JohanEngelen** [comment]
let me know where the right place for PRs is @ljmf00   (here or in weka fork)  thnx :)

File: source/ldclint/visitors.d [hunk 1]
```diff
@@ -26,7 +26,10 @@ import dmd.nspace;
 import dmd.init;
 import dmd.cond;
 
-import dmd.root.rootobject;
+static if (__VERSION__ >= 2108)
+    import dmd.rootobject;
+else
+    import dmd.root.rootobject;
```
Comment (ljmf00-wekaio) [inline source/ldclint/visitors.d:32]: I have a bridge set of modules for these situations at `source/ldclint/dmd`. I can fix it later.

**Review by ljmf00-wekaio — APPROVED** [review]

**Comment by ljmf00-wekaio** [comment]
> let me know where the right place for PRs is @ljmf00 (here or in weka fork) thnx :)

here, but weka repo should be in-sync if we want to have it used in weka's CI, since only `weka` repos are pulled there.

**Comment by JohanEngelen** [comment]
good to merge from my side

**Comment by ljmf00-wekaio** [comment]
Sorry, I wanted to squash the commit message to use conventional commits (https://www.conventionalcommits.org/en/v1.0.0/).

**Comment by ljmf00-wekaio** [comment]
Merged and synced with weka repo.

**Pull request merged**